pkg/envoy_access_log_server/service_v3.go

```diff
@@ -0,0 +1,202 @@
+		if t, err := ptypes.Timestamp(ts); err == nil {
+			result["startTime"] = t.Format(time.RFC3339)
+		}
+	}
+	result["timeToFirstDownstreamTxByte"] = pbDurationDeterministicString(common.TimeToFirstDownstreamTxByte)
+	result["timeToFirstUpstreamRxByte"] = pbDurationDeterministicString(common.TimeToFirstUpstreamRxByte)
+	result["timeToFirstUpstreamTxByte"] = pbDurationDeterministicString(common.TimeToFirstUpstreamTxByte)
+	result["timeToLastDownstreamTxByte"] = pbDurationDeterministicString(common.TimeToLastDownstreamTxByte)
+	result["timeToLastRxByte"] = pbDurationDeterministicString(common.TimeToLastRxByte)
+	result["timeToLastUpstreamRxByte"] = pbDurationDeterministicString(common.TimeToLastUpstreamRxByte)
+	result["timeToLastUpstreamTxByte"] = pbDurationDeterministicString(common.TimeToLastUpstreamTxByte)
+	result["upstreamCluster"] = common.UpstreamCluster
+	if common.UpstreamLocalAddress != nil {
+		if sa := common.UpstreamLocalAddress.GetSocketAddress(); sa != nil {
+			result["upstreamLocalAddress"] = sa.GetAddress()
+			result["upstreamLocalPort"] = fmt.Sprint(sa.GetPortValue())
+		}
+	}
+	if common.UpstreamRemoteAddress != nil {
+		if sa := common.UpstreamRemoteAddress.GetSocketAddress(); sa != nil {
+			result["upstreamRemoteAddress"] = sa.GetAddress()
+			result["upstreamRemotePort"] = fmt.Sprint(sa.GetPortValue())
+		}
+	}
+	result["upstreamTransportFailureReason"] = common.UpstreamTransportFailureReason
+	result["sampleRate"] = strconv.FormatFloat(common.SampleRate, 'f', -1, 64)
+
+	return result
+}
+
+func exportHTTPRequestPropertiesV3(request *envoy_data_accesslog_v3.HTTPRequestProperties) stringmap.StringMap {
+	result := stringmap.StringMap{}
+
+	result["authority"] = request.Authority
+	result["forwardedFor"] = request.ForwardedFor
+	result["originalPath"] = request.OriginalPath
+	result["path"] = request.Path
+	result["port"] = strconv.FormatUint(uint64(request.Port.GetValue()), 10)
+	result["referer"] = request.Referer
+
+	// request headers are encoded with `http_` prefix (to keep the scheme used by Nginx)
+	if request.RequestHeaders != nil {
+		for header := range request.RequestHeaders {
+			result["http_"+header] = request.RequestHeaders[header]
+		}
+	}
+
+	result["requestBodyBytes"] = strconv.FormatUint(request.RequestBodyBytes, 10)
+	result["requestHeadersBytes"] = strconv.FormatUint(request.RequestHeadersBytes, 10)
+	result["requestId"] = request.RequestId
+	result["requestMethod"] = request.RequestMethod.String()
+	result["scheme"] = request.Scheme
+	result["userAgent"] = request.UserAgent
+
+	return result
+}
+
+func exportHTTPResponsePropertiesV3(response *envoy_data_accesslog_v3.HTTPResponseProperties) stringmap.StringMap {
+	result := stringmap.StringMap{}
+
+	result["responseBodyBytes"] = strconv.FormatUint(response.ResponseBodyBytes, 10)
+	result["responseCodeDetails"] = response.ResponseCodeDetails
+	result["responseCode"] = strconv.FormatUint(uint64(response.ResponseCode.GetValue()), 10)
```

Comment:
not mandatory, just thinking:

Do you agree to change name of `statuCode` in https://github.com/seznam/slo-exporter/blob/master/examples/all_in_one/slo-exporter/conf/slo_exporter.yaml#L15  to `responseCode`? What name is better? Ideally we should use the same naming.